wandb/cli/cli.py

```diff
@@ -650,7 +650,31 @@ def _clean():
 @click.option("--program", default=False, help="Set sweep program")
 @click.option("--settings", default=False, help="Set sweep settings", hidden=True)
 @click.option("--update", default=None, help="Update pending sweep")
-@click.argument("config_yaml")
+@click.option(
+    "--stop",
+    is_flag=True,
+    default=False,
+    help="Finish a sweep to stop running new runs and let currently running runs finish.",
+)
+@click.option(
+    "--cancel",
+    is_flag=True,
+    default=False,
+    help="Cancel a sweep to kill all running runs and stop running new runs.",
+)
+@click.option(
+    "--pause",
+    is_flag=True,
+    default=False,
+    help="Pause a sweep to temporarily stop running new runs.",
```

Comment:
```suggestion
    help="Pause a sweep: temporarily stop running new runs.",
```

What is the difference between stop and pause?

Reply:
When you stop a sweep, the agents running that sweep terminates. When you pause a sweep, the agents keep waiting for you to resume the sweep.

Reply:
Can we add a description of that distinction to the CLI help here.

Reply:
Pause a sweep. This command instructs the sweep agent process to stop requesting new work. This command can be undone with "resume."